app/client/components/dashboard/beneficiaryLiveFeed.jsx

```diff
@@ -61,9 +61,15 @@ class BeneficiaryLiveFeed extends React.Component {
                   typeof transfer.recipient_user !== "undefined"
                 ) {
                   var recipient_user = users.byId[transfer.recipient_user];
-                  var recipient_user_name =
-                    recipient_user.first_name + " " + recipient_user.last_name;
-                } else if (recipient_blockchain_address !== null) {
+                  if (typeof recipient_user !== "undefined") {
+                    var recipient_user_name =
+                      recipient_user.first_name +
+                      " " +
+                      recipient_user.last_name;
+                  }
```

Comment:
`recipient_user` was sometimes undefined, causing a bug in sentry https://sentry.io/organizations/sempo/issues/1580313759/?project=1228848&query=is%3Aunresolved